bin/sandbox

```diff
@@ -3,6 +3,10 @@
 # Used in the sandbox rake task in Rakefile
 
 set -e
+if [ ! -z $DEBUG ]
```

Comment:
Not a big deal, but I think it won't catch when `$DEBUG` is set to the empty string:

```bash
> [ ! -z '' ]
> echo $?
1
```

I've usually seen to check for the presence of an env variable is `if [ -z ${var+x} ]`, which returns false if the variable is set to the empty string or any other string.

Reply:
I think it's ok to assume it will have some contents when set, that said maybe I'm doing some bash mistake here:

```
solidus:elia/frontend-templates ⤑ bash -l -c 'DEBUGz="asdf" test -z ${DEBUG+x}; echo $?'
0
solidus:elia/frontend-templates ⤑ bash -l -c 'DEBUG="asdf" test -z ${DEBUG+x}; echo $?'
0
solidus:elia/frontend-templates ⤑ bash -l -c 'DEBUG="" test -z ${DEBUG+x}; echo $?'
0
```

Reply:
It looks like the parameter expansion is evaluated before the command, so $DEBUG is never defined at that point. But if you run it as part of a script, it'll work.

Reply:
Thanks for the explanation, let me know if you would still prefer to support an empty string there

Reply:
I think it makes sense to support them. I know the code looks weirder, but it's not our fault that bash is bizarre 😄 Up to you! 🙂

Reply:
Done! TIL 🙏